storage/rocksdb/rdb_datadic.h

```diff
@@ -409,7 +432,11 @@ class Rdb_key_def {
     //    an inefficient where data that was a multiple of 8 bytes in length
     //    had an extra 9 bytes of encoded data.
     SECONDARY_FORMAT_VERSION_UPDATE2 = 12,
-    SECONDARY_FORMAT_VERSION_LATEST = SECONDARY_FORMAT_VERSION_UPDATE2,
+    // This change includes support for covering SK lookups for varchars
+    //  - A 2-byte bitmap is added after the header to unpack_info only for
+    //    records which have covered varchar columns.
+    SECONDARY_FORMAT_VERSION_UPDATE3 = 13,
+    SECONDARY_FORMAT_VERSION_LATEST = SECONDARY_FORMAT_VERSION_UPDATE3,
```

Comment:
I don't know if we want to make the latest version default yet. Maybe we should wait until we at least get blob/text support?